src/Sylius/Bundle/AdminBundle/Resources/views/Security/login.html.twig

```diff
@@ -9,7 +9,7 @@
 {% block body %}
     <div class="page page-center">
         <div class="container container-tight py-4">
-            {% hook main_hook with { parent_main_hook: main_hook, form } %}
+            {% hook main_hook with { parent_main_hook: main_hook, form, last_error, last_username } %}
```

Comment:
```suggestion
            {% hook main_hook with hookable_data|merge({ parent_main_hook: main_hook, form, last_error, last_username }) %}
```
or is there a reason why you didn't use this method?

Reply:
It is an entry template, so there's no `hookable_data`.